docs/dqx/docs/reference/quality_checks.mdx

```diff
@@ -28,8 +28,8 @@ You can also define your own custom checks (see [Creating custom checks](#creati
-| `is_not_null_and_is_in_list`       | Checks whether the values in the input column are not null and present in the list of allowed values. This check is not suited for large lists of allowed values. In such cases, it’s recommended to use the `foreign_key` dataset-level check instead.                                                                                                                                                                           | `column`: column to check (can be a string column name or a column expression); `allowed`: list of allowed values                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `is_in_list`                       | Checks whether the values in the input column are present in the list of allowed values (null values are allowed). We can pass an additional case_sensitive parameter as False for a case insensitive check. This check is not suited for large lists of allowed values. In such cases, it’s recommended to use the `foreign_key` dataset-level check instead.                                                                                                                                                              | `column`: column to check (can be a string column name or a column expression); `allowed`: list of allowed values                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Please follow the format of other check. Add the new param to the Arguments column, instead of describing it in the Description